flowkit/go.mod

```diff
@@ -180,3 +180,5 @@ require (
 	modernc.org/memory v1.5.0 // indirect
 	modernc.org/sqlite v1.21.1 // indirect
 )
+
+replace github.com/onflow/flow-go-sdk => github.com/findonflow/flow-go-sdk v0.40.1-0.20230717122814-47abbbe6e6b0
```

Comment:
why is this needed?

Reply:
because at this time the pr needed a new version of flow-go-sdk since it added a new method to it

Reply:
https://github.com/onflow/flow-go-sdk/pull/427 this pr is still not merged

Reply:
merged the PR and released a new version https://github.com/onflow/flow-go-sdk/releases/tag/v0.42.0

Reply:
fixed